drake/systems/plants/RigidBodySystem.h

```diff
@@ -142,13 +142,13 @@ class DRAKERBSYSTEM_EXPORT RigidBodySystem {
   RigidBodySystem(const std::shared_ptr<RigidBodyTree>& rigid_body_tree)
       : tree(rigid_body_tree),
         use_multi_contact(false),
-        penetration_stiffness(150.0),
+        penetration_stiffness(50.0),
```

Comment:
these numbers were arbitrary enough, but why change the defaults (instead of setting them in your app?)